frontend/src/employee-frontend/components/messages/MessagesPageContext.tsx

```diff
@@ -0,0 +1,171 @@
+import React, {
+  createContext,
+  useCallback,
+  useEffect,
+  useMemo,
+  useState
+} from 'react'
+import { Loading, Paged, Result } from '../../../lib-common/api'
+import { useRestApi } from '../../../lib-common/utils/useRestApi'
+import { UUID } from '../../types'
+import { getMessageDrafts, getReceivedMessages, getSentMessages } from './api'
+import { DraftContent, MessageThread, SentMessage } from './types'
+import { AccountView } from './types-view'
```

Comment:
Stuff from `types-view.d.ts` could be moved to `types.ts`, because they cross-import each other.

Reply:
It's 2021 and there's no built-in tool for checking circular dependencies ☹️  Moved the view-related message boxes to `types-view`.